pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md

```diff
@@ -212,6 +212,55 @@ velero
  ✓ Ensuring check that there remain no addons and deletion of the Kubeaddons controller
 ```
 
+If there is a timeout error at this step, start `kommander migrate -y` again, and it will eventually continue where this timed out.
+
+When this is complete, you will have to edit a Traefik Middleware in order for Gitea to correctly perform.
+
+To do this, edit the Middleware object:
```

Comment:
Maybe change this to modify `traefik-10.3.0-migration-overrides` configmap to make it a safer bet?